QuestionSets/PythonLanguage/ClassesAndObjects/Rajat_Varma/Answer_1.py

```diff
@@ -0,0 +1,68 @@
+class Player:
+    def __init__(self):
+        age = int()
+        movement_speed = int()
+        health = int()
+        money = int()
+
+        self.age = age
+        self.movement_speed = movement_speed
+        self.health = health
+        self.money = money
+
```

Comment:
Why wouldn't you just assign `self.age = int()` , `self.movement_speed = int()`, and so on. 
The extra variables on lines 3 - 6 are a bit redundant.